src/actions/ProjectActions.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash');
+
+var AppDispatcher = require('../dispatcher/AppDispatcher');
+var ProjectConstants = require('../constants/ProjectConstants');
+var defaults = require('../config').defaults;
+
+var ProjectActions = {
+  create: function() {
+    AppDispatcher.dispatch({
+      actionType: ProjectConstants.PROJECT_CREATED,
+      project: _.cloneDeep(defaults)
```

Comment:
This isn’t the right place to create the project—should happen in the `ProjectStore`